sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/init.sql

```diff
@@ -0,0 +1,634 @@
+--Query to initialize clients_first_seen_v2
+-- Earliest ping dates for each ping used in the query.
+DECLARE earliest_new_profile_date DATE DEFAULT '2017-06-26';
+
+DECLARE earliest_main_date DATE DEFAULT '2016-03-12';
+
+DECLARE earliest_shutdown_date DATE DEFAULT '2018-10-30';
+
+-- Starting date to query pings.
+DECLARE starting_date DATE DEFAULT '2020-01-01';
```

Comment:
The current start date for clients_first_seen_v1 is '2020-01-01'. Documenting the available dates above in case older data is required.

Reply:
Talking to George, we will update all available data to support acquisition analysis.